apps/mudita-center-e2e/src/specs/overview/kompakt-overview.ts

```diff
@@ -0,0 +1,140 @@
+
+describe("E2E mock sample - overview view", () => {
+  before(async () => {
+    E2EMockClient.connect()
+    //wait for a connection to be established
+    await browser.waitUntil(() => {
+      return E2EMockClient.checkConnection()
+    })
+  })
+
+  after(() => {
+    E2EMockClient.stopServer()
+    E2EMockClient.disconnect()
+  })
+
+  it("Connect device", async () => {
+    E2EMockClient.mockResponse({
+      path: "path-1",
+      body: overviewDataWithOneSimCard,
+      endpoint: "FEATURE_DATA",
+      method: "GET",
+      status: 200,
+    })
+    E2EMockClient.addDevice({
+      path: "path-1",
+      serialNumber: "first-serial-number",
+    })
+
+    await browser.pause(6000)
+    const menuItem = await $(`//a[@href="#/generic/mc-overview"]`)
+
+    await menuItem.waitForDisplayed({ timeout: 10000 })
+    await expect(menuItem).toBeDisplayed()
+  })
+
+  it("Overwrite device response", async () => {
+    // const badge = $(`//p[contains(text(), 'Offline')]`)
+
+    // await badge.waitForDisplayed()
+    // await expect(badge).toBeDisplayed()
+
+    // // overwrite default response for given device
+    // E2EMockClient.mockResponse({
+    //   path: "path-1",
+    //   body: overviewDataWithOneSimCard,
+    //   endpoint: "FEATURE_DATA",
+    //   method: "GET",
+    //   status: 200,
+    // })
+
+    // // overwrite newest response for given device
+    // E2EMockClient.mockResponseOnce({
+    //   path: "path-1",
+    //   body: outboxReloadOverview,
+    //   endpoint: "OUTBOX",
+    //   method: "GET",
+    //   status: 200,
+    // })
+
+    // await badge.waitForDisplayed({ reverse: true })
+    // screenshotHelper.makeViewScreenshot()
+    // await expect(badge).not.toBeDisplayed()
+    await browser.pause(10000)
+  })
+
+  it("Verify Overview Page", async () => {
+    const muditaOSVersion = await OverviewPage.muditaOSVersion
+    await expect(muditaOSVersion).toBeDisplayed
+
+    const serialNumberLabel = OverviewKompaktPage.serialNumberLabel
+    const serialNumberValue = OverviewKompaktPage.serialNumberValue
+    await expect(serialNumberLabel).toHaveText("Serial number")
+    await expect(serialNumberValue).toHaveText("0123456789ABCDEF")
+
+    const aboutYourDevice = OverviewKompaktPage.aboutYourDevice
+    await expect(aboutYourDevice).toBeDisplayed
+
+    const checkSARInformationButton = OverviewPage.checkSARInformationButton
+    await expect(checkSARInformationButton).toBeDisplayed
+
+    const createBackupButton = OverviewPage.createBackupButton
+    await expect(createBackupButton).toBeDisplayed
+
+    const backupInfo = OverviewKompaktPage.backupInfo
+    await expect(backupInfo).toBeDisplayed
+    await expect(backupInfo).toHaveTextContaining(
+      "You haven’t backed up your device yet"
+    )
+    //not full text to be checked due to spacing issue
+
+    const kompaktImageElement = overviewKompaktPage.kompaktImage
+    await expect(kompaktImageElement).toBeDisplayed
```

Comment:
You need to verify if image is correct in `src` - white or black (black for now)